### PR TITLE
adds a tactical nail gun to CE’s vendor

### PR DESCRIPTION
**Pull request by deathrobotpunch**
# About the pull request

adds a tactical nail gun to CE’s vendor.

# Explain why it's good for the game
CE should have one of these already, CE’s not having sentries is understandable but they should have basic tools available to them.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
balance: The Tactical compact nail gun is now available in the Chief Engineers vendor. 
/:cl:

### CODE REVIEW

**Comment by Nanu308** [comment]
This is not a QoL change, this is a Balance change, albeit a minor one.

File: code/game/machinery/vending/vendor_types/crew/senior_officers.dm [hunk 1]
```diff
@@ -90,6 +90,7 @@ GLOBAL_LIST_INIT(cm_vending_clothing_chief_engineer, list(
 		list("Marine Combat Boots", 0, /obj/item/clothing/shoes/marine, MARINE_CAN_BUY_SHOES, VENDOR_ITEM_MANDATORY),
 		list("Welding Goggles", 0, /obj/item/clothing/glasses/welding, MARINE_CAN_BUY_GLASSES, VENDOR_ITEM_MANDATORY),
 		list("KN5500/2 PDA", 0, /obj/item/device/working_joe_pda/uscm, MARINE_CAN_BUY_KIT, VENDOR_ITEM_MANDATORY),
+		list("KN5500/2 PDA", 0, /obj/item/weapon/gun/smg/nailgun/compact/tactical, MARINE_CAN_BUY_KIT, VENDOR_ITEM_MANDATORY),
```
Comment (Drulikar) [inline code/game/machinery/vending/vendor_types/crew/senior_officers.dm:93]: Please fix the item name in the vendor. It is not a PDA.

**Review by Drulikar — CHANGES_REQUESTED** [review]

File: code/game/machinery/vending/vendor_types/crew/senior_officers.dm (hunk 1 above)
Comment (deathrobotpunch) [reply code/game/machinery/vending/vendor_types/crew/senior_officers.dm:93]: my fault!

**Review by Drulikar — APPROVED** [review]

**Comment by Drulikar** [comment]
See https://github.com/cmss13-devs/cmss13/issues/7465
as a result of `MARINE_CAN_BUY_KIT` if your intention was for both to be purchaseable together.

**Comment by nuklearcellphoneg** [comment]
The nailgun comes with no nail magazine, and there's no way for CE to purchase nails.

**Pull request merged**